src/test/resources/journeys.conf

```diff
@@ -14,18 +14,39 @@
 
 journeys {
 
-  uknw-auth-checker-api = {
-    description = "UKNW Auth Checker API Journey"
-    load = 0.01
-    parts = [
-      AuthJourney
-      UknwAuthCheckerApiJourney
-    ]
+  uknw-auth-checker-single-eori = {
+    description = "UKNW Auth Checker Single EORI Journey"
+    load = 0.001
+    parts = [SingleEoriJourney]
   }
+  uknw-auth-checker-100-eori = {
+      description = "UKNW Auth Checker 100 EORI Journey"
+      load = 0.001
```

Comment:
Please can you update the load to what's described per journey on here: https://confluence.tools.tax.service.gov.uk/display/NOP/Performance+testing

Reply:
Changed load as mentioned in documentation.